src/Lookup/LabelLookup.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Lookup;
+
+use Wikibase\DataModel\Entity\EntityId;
+use Wikibase\DataModel\Term\Term;
+
+/**
+ * @since 3.9
+ *
+ * @license GPL-2.0+
+ * @author Thiemo Mättig
+ */
+interface LabelLookup {
+
+	/**
+	 * @param EntityId $entityId
+	 *
+	 * @throws LabelDescriptionLookupException
+	 * @return Term|null
```

Comment:
Not an issue with this PR, just an FYI: looking at this code after having been a run away'd from Wikibase for some time now, I find it weird this type is Term and not Label.

Reply:
There is no Label class, but a common Term class for both labels and descriptions.